packages/utilities/src/IFabricConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { IStyleSheetConfig } from '@uifabric/merge-styles';
```

Comment:
moved this here out of the styling package, since it's no longer exclusively used by styling

also added `theme` to it since theme.ts was using `theme` off of it anyways